app/uk/gov/hmrc/incorporatedentityidentificationfrontend/connectors/CreateBusinessVerificationJourneyConnector.scala

```diff
@@ -46,7 +46,8 @@ class CreateBusinessVerificationJourneyConnector @Inject()(http: HttpClient,
           "ctUtr" -> ctutr
         )),
         "continueUrl" -> routes.BusinessVerificationController.retrieveBusinessVerificationResult(journeyId).url,
-        "accessibilityStatementUrl" -> journeyConfig.pageConfig.accessibilityUrl
+        "accessibilityStatementUrl" -> journeyConfig.pageConfig.accessibilityUrl,
+        "deskproServiceName" -> journeyConfig.pageConfig.optServiceName
```

Comment:
I think deskproServiceName should be deskProServiceId from our page config. optServiceName maps to pageTitle in the Business Verification config

Reply:
done